0.5/articles/unit-testing-elements.md

```diff
@@ -240,7 +240,7 @@ We begin writing our first test with a new HTML file in the "tests" directory of
 
 * An import for our element (core-selector)
 
-We can repurpose the existing "basic-test.html" file for this purpose. Let's rename it to "core-selector-tests.html" and customize it for `<core-selector>`:
+We can repurpose the existing "basic-test.html" file for this purpose. Let's rename it to "core-selector-tests.html" and customize it for `<core-selector>` (? has it been changed?):
```

Comment:
Is it changed? ...or you're just quoting the file contents *before* it's changed?